andriy.safronov/03.rb

```diff
@@ -0,0 +1,34 @@
+class Tamagochi
+    def status 
+        "Поїсти: #{@eat_time}, Попити: #{@drink_time}, Спати: #{@sleep_time}"
+    end 
+    def initialize
+        @eat_time = 8
```

Comment:
indent should be two spaces. You can read this ruby style guide https://github.com/github/rubocop-github/blob/master/STYLEGUIDE.md